File: hub-kit-backend/src/main/java/com/inetum/realdolmen/hubkitbackend/dto/MotorDTO.java
package com.inetum.realdolmen.hubkitbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class MotorDTO extends VehicleDTO {
    private String markType;
}

File: crash-kit-app/app/src/main/java/com/inetum/realdolmen/crashkit/dto/MotorDTO.kt
package com.inetum.realdolmen.crashkit.dto

data class MotorDTO(
    @Transient override val id: Int?,
    @Transient override val licensePlate: String,
    @Transient override val countryOfRegistration: String,
    var markType: String
) : Vehicle(id, licensePlate, countryOfRegistration)
